src/addons/__tests__/update-test.js

```diff
@@ -72,7 +72,10 @@ describe('update', function() {
   });
 
   it('should support apply', function() {
-    expect(update(2, {$apply: function(x) { return x * 2; }})).toEqual(4);
+    function f(x) {
```

Comment:
eslint requires indenting code blocks, instead of doing them on the same line. This is part of [`brace-style`](http://eslint.org/docs/rules/brace-style). If you'd prefer to allow single-line function declarations, we can enable `allowSingleLine`.